### DISCUSSION TITLE
How to overwrite previous release and tag for early-access on GitHub

### DISCUSSION THREAD

**Discussion by ayechantharsoe** [jreleaser/jreleaser#325] [General]
If current version is a SNAPSHOT, I believe early-access release and associated tag is created/updated each time a commit/pr is made to main branch.

However, this is not the case for my maven project. 
Every time I merge a PR, it creates a new release and also the tag is not created as mentioned in docs using current ${project.version}.

```
 <!--
        The tag associated with the release.
        May define a `JRELEASER_TAG_NAME` environment variable instead.
        If left unspecified, will use `v{{projectVersion}}`.
          
      -->
      <tagName>v1.0.0</tagName>
```

My pom.xml configuration for GitHub release is following.
```
    <release>
        <github>
            <owner>venustulips</owner>
            <overwrite>true</overwrite>
            <update>true</update>
            <connectTimeout>60</connectTimeout>
            <username>venustulips</username>
            <changelog>
                // removed
            </changelog>
        </github>
    </release>
```

I am not creating an issue yet because I believe I am missing to correct some configuration.

![release](https://user-images.githubusercontent.com/8254188/127607133-040423c3-6744-4f31-8c98-602e327b862b.png)

Regards

**Comment by aalmiray** [comment]
@venustulips I've noticed that from time to time the JReleaser early-access releases are also placed on draft mode when they should had been published as normal. Could you run `mvn jreleaser:config` and inspect the output of the `release:` block?

**Reply by ayechantharsoe** [reply]
I just noticed that multiple early access releases happens only when merging a PR to main branch. 
The issue does not happens with the commit made directly to main branch.

**Reply by ayechantharsoe** [reply]
However, I do not find any difference between release config logs of direct commit and pr merge commit.
I checked release config logs printed to console from GitHub Action.
